components/responsive-wrapper.tsx

```diff
@@ -0,0 +1,45 @@
+import { ReactNode } from "react";
+
+import clsx from "clsx";
+
+interface ResponsiveWrapperProps {
+  xs?: boolean;
+  sm?: boolean;
+  md?: boolean;
+  lg?: boolean;
+  xl?: boolean;
+  xxl?: boolean;
+  children?: ReactNode;
+}
+
+export default function ResponsiveWrapper({
+  xs,
```

Comment:
if we end up using `useBreakPoint` from the other comment, this becomes easier, as we don't need to rely on booleans and we can simply use the value from `currentSize`